OPHD/Things/Structures/Structure.h

```diff
@@ -63,6 +63,7 @@ class Structure: public Thing
 
 public:
 	Structure(const std::string& name, const std::string& spritePath, StructureClass structureClass);
+	virtual ~Structure() {}
```

Comment:
I think we want virtual destructors in both Structure and PowerStructure since they are base classes?

Reply:
Yes, but technically they already have a virtual destructor inherited from `Thing`:
```cpp
	virtual ~Thing()
	{
		#ifdef _DEBUG
		std::cout << mName << ": He's dead Jim!" << std::endl;
		#endif
	}
```

As such, you don't need to explicitly mention it in derived classes, unless you're creating a specific `override` for it.

Reply:
Ah right, I missed checking Thing. Thanks.

Reply:
Still makes sense to declare it explicitly as a method of documentation.

Reply:
Mmm, I find it mildly confusing when an explicit override is mentioned, but does nothing. It makes me question what's going on, and I generally go digging to find an implementation, which might not actually exist in any meaningful way. I find it less confusing when all mention is simply dropped.

Plus, given things like `std::unique_ptr`, and it's ability to be used as a base for RAII type objects, very few of our own should have explicit destructors. There are a number of classes whose destructors I would like to optimize away.